huxley/www/js/components/Huxley.js

```diff
@@ -51,21 +52,30 @@ var Huxley = React.createClass({
     var user = CurrentUserStore.getCurrentUser();
     if (User.isAnonymous(user)) {
       return (
+        <div>
```

Comment:
Adjacent jsx elements have to be wrapped in divs otherwise it can cause a runtime error. Not really sure why that happens but its apparently a known problem